.github/workflows/release.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           distribution: temurin
           java-version: 11
           check-latest: true
+      - name: Set release version
+        run: |
+          release_version=$(git describe --tags --abbrev=0 --exact-match)
+          export ORG_GRADLE_PROJECT_releaseVersion="${release_version:1}"
```

Comment:
Is the `:1` a default? Would that cause issues (i.e., a bad publication) if it was used?

Reply:
I cut off the first char cause it's always `v`. The convention for publishing is to do `v*` but we strip the v in the publish version

Reply:
First char of what? I'm asking specifically about the `:1` that is in quotes. What does that do?

Reply:
Ah, substring? https://stackoverflow.com/a/15143895